Operator/src/main/java/io/streamzi/ev/Manager.java

```diff
@@ -3,22 +3,21 @@
 import io.streamzi.ev.operator.ConfigMapOperator;
 import io.streamzi.ev.operator.DeploymentConfigOperator;
 import io.streamzi.ev.operator.DeploymentOperator;
-import io.streamzi.ev.watcher.DeploymentConfigWatcher;
 import io.streamzi.ev.watcher.ConfigMapWatcher;
+import io.streamzi.ev.watcher.DeploymentConfigWatcher;
 import io.streamzi.ev.watcher.DeploymentWatcher;
+import org.apache.logging.log4j.LogManager;
 
 import java.util.concurrent.ExecutorService;
 import java.util.concurrent.Executors;
 import java.util.concurrent.TimeUnit;
-import java.util.logging.Level;
-import java.util.logging.Logger;
 
 /**
  * Start the ConfigMap Watcher
  */
 public class Manager {
 
-    private static final Logger logger = Logger.getLogger(Manager.class.getName());
+    private final static org.apache.logging.log4j.Logger logger = LogManager.getLogger(Manager.class);
```

Comment:
why the full package?

Reply:
Didn't spot that - IDE artefact when changing to log4j. Will fix.